can_tools/scrapers/official/MO/mo_state.py

```diff
@@ -0,0 +1,233 @@
+from abc import ABC
+
+import pandas as pd
+
+import pyppeteer
+import us
+from typing import List
+
+from can_tools.scrapers.base import CMU
+from can_tools.scrapers.official.base import StateDashboard
+from can_tools.scrapers.puppet import TableauNeedsClick
+
+
+class MissouriBase(
+    StateDashboard,
+    TableauNeedsClick,
+    ABC,
+):
+    source = "https://results.mo.gov/t/COVID19/views/COVID-19DataforDownload/{tab}?:isGuestRedirectFromVizportal=y&:embed=y"
+    has_location = False
+    location_type = "county"
+    state_fips = int(us.states.lookup("Missouri").fips)
+
+    out_cols = [
+        "dt",
+        "vintage",
+        "location_name",
+        "category",
+        "measurement",
+        "unit",
+        "age",
+        "race",
+        "sex",
+        "value",
+    ]
+
+    # def clean_desoto(self, df: pd.DataFrame):
+    #    df.loc[df["location_name"] == "Desoto", "location_name"] = "DeSoto"
+
+
+async def _special_tableau_handler(page, tmpdirname):
+    # Get a reference to the page that was just opened
+    pages = await page.browser.pages()
+    popup = pages[-1]
+
+    import requests
```

Comment:
Sorry about that. Agreed. This code is no where near ready, it is just calling code for the puppet. Will move them up.